PyTables expects that LIBS is a list of library names, and will add '-l' by itself.
original patch by Ward Poelmans (VUB), ported to PyTables 3.5.2 by Kenneth Hoste (HPC-UGent)
--- PyTables-3.5.2/setup.py.orig	2020-04-23 10:47:42.558725717 +0200
+++ PyTables-3.5.2/setup.py	2020-04-23 10:50:20.163750135 +0200
@@ -478,6 +478,9 @@
     # is not a good idea.
     CFLAGS = os.environ.get('CFLAGS', '').split()
     LIBS = os.environ.get('LIBS', '').split()
+    for idx, lib in enumerate(LIBS):
+        if lib.startswith("-l"):
+            LIBS[idx] = lib.replace("-l", "")
     CONDA_PREFIX = os.environ.get('CONDA_PREFIX', '')
     # We start using pkg-config since some distributions are putting HDF5
     # (and possibly other libraries) in exotic locations.  See issue #442.
